src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,19 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from '../TodoComponents/Todo'
+
+const TodoList = props => {
+    return(
+        <div>
+        {
+            props.todo.map(ele => {
```

Comment:
You could use filter here to implement search.  You'll get opportunities to do that next week.